x-pack/legacy/plugins/uptime/server/graphql/monitors/schema.gql.ts

```diff
@@ -120,6 +120,11 @@ export const monitorsSchema = gql`
     monitors: [LatestMonitor!]
   }
 
+  type GetHistogramResult {
```

Comment:
This name is a bit awkward. WDYT of `HistogramResult` instead?

Reply:
Yes that's better